### PR TITLE
feat: Add Malayalam language [Translation]

### PR DESCRIPTION
**Pull request by Justinjdaniel**
Hi,
This pull request adds a complete translation for Malayalam (ML) version v1.0.0.

### CODE REVIEW

**Comment by damianopetrungaro** [comment]
Hello @Justinjdaniel sorry for the delay!
I see it is missing an update on the configuration file as well as stated in the readme, would you mind adding it?

Once you are done ping me so we can merge this!

**Comment by Justinjdaniel** [comment]
Hello @damianopetrungaro 
As I was updating the config file, I discovered that the branch was not synchronized and had a tiny problem, so I corrected the weight value for the order to be accurate, but there is still a conflict to resolve, so I hope everything is good.

Thanks.

File: config.yaml [hunk 1]
```diff
@@ -345,7 +346,6 @@ languages:
     versions:
       current: v1.0.0
       list:
-      - v1.0.0
```
Comment (damianopetrungaro) [inline config.yaml:348]: Why did you remove this line?

Comment (Justinjdaniel) [reply config.yaml:348]: Sorry, I accidentally missed that part while updating the `config.yaml` and fixed it.

Thanks

**Comment by bcoe** [comment]
@damianopetrungaro is this looking good now?

**Comment by damianopetrungaro** [comment]
Looks good indeed!

**Review by damianopetrungaro — APPROVED** [review]

**Comment by cshintov** [comment]
Hi @Justinjdaniel How did you generate this? Some kind of LLM? Or was it manually translated by you?
